src/Utils/dispatchUtils.ts

```diff
@@ -198,6 +194,22 @@ function associateModelDialogsWithDispatchActionsAndClean(sourceModelPairs: Sour
     })
 }
 
+const generateDispatchDialog = (rounds: CLM.TrainRound[]) => (
```

Comment:
What is the warning that caused you to move this? These types of edits are difficult to review. Reviewer can't tell if it's moved or moved _and_ edited etc and try to avoid these operations unless it's following some standard. Lars was on a bit of moving all private methods below the `render` function on all React components recently because some warning said public before private which didn't make sense for components as you want to be able to scroll to bottom and find the JSX.

Reply:
I don't feel strongly about having privates at the bottom.  We could change the tslib to remove this warning.

Reply:
This was to address the "use before declaration" warning.